client/src/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+export {Home} from "./Home/Home"

```

Comment:
Same as above
Always remember to run `yarn format:write`

Reply:
ok